unlock-app/src/components/interface/verifiers/VerifiersList.tsx

```diff
@@ -137,11 +126,17 @@ export const VerifiersList: React.FC<VerifiersListProsps> = ({
         <tbody>
           {verifiers?.map((verifier: any, index: number) => {
             const key = `${verifier?.address}-${index}`
+            const createdAt = verifier?.createdAt
+              ? new Date(verifier?.createdAt).toLocaleString()
+              : '-'
+            const verifiedAt = verifier?.verifiedAt
```

Comment:
What is this? I don't think we need it here.

Reply:
is `updatedAt`